xbmc/utils/FileUtils.h

```diff
@@ -28,4 +35,9 @@ class CFileUtils
   */
   static CDateTime GetModificationDate(const std::string& strFileNameAndPath, const bool& bUseLatestDate);
   static CDateTime GetModificationDate(const int& code, const std::string& strFileNameAndPath);
+  /*! \brief Get the creation date of a file if possible
+  \param strFileNameAndPath path to the file
+  \return Returns the creation date, else returns the earlier of the files ctime and mtime if valid
+  */
+  static CDateTime GetCreationDate(const std::string & strFileNameAndPath);
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types